LiverMarkups/VTKWidgets/vtkSlicerBezierSurfaceRepresentation3D.cxx

```diff
@@ -101,6 +105,26 @@ vtkSlicerBezierSurfaceRepresentation3D::vtkSlicerBezierSurfaceRepresentation3D()
   this->BezierSurfaceNormals = vtkSmartPointer<vtkPolyDataNormals>::New();
   this->BezierSurfaceNormals->SetInputConnection(this->BezierSurfaceSource->GetOutputPort());
 
+  this->BezierPlane = vtkSmartPointer<vtkBezierSurfaceSource>::New();
+  this->BezierPlane->SetResolution(20,20);
+  auto PlaneControlPoints = vtkSmartPointer<vtkPoints>::New();
+
+  for(int i=0;i<4;i++){
+    PlaneControlPoints->InsertNextPoint(-60,(i*40),0);
+    PlaneControlPoints->InsertNextPoint(-20,(i*40),0);
+    PlaneControlPoints->InsertNextPoint(20,(i*40),0);
+    PlaneControlPoints->InsertNextPoint(60,(i*40),0);
+
+    }
```

Comment:
Why are these values set to 60, 40, etc? If these are for the placement of the resectogram overlay, we should use normalized device coordinates. Here is an example: [https://kitware.github.io/vtk-examples/site/Cxx/Annotation/MultiLineText/](https://kitware.github.io/vtk-examples/site/Cxx/Annotation/MultiLineText/)